de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/negotiation/AbstractIncomingProjectNegotiation.java

```diff
@@ -236,13 +240,13 @@ protected void cleanup(IProgressMonitor monitor,
         fileReplacementInProgressObservable.replacementDone();
 
         /*
-         * TODO Queuing responsibility should be moved to Project
-         * Negotiation, since its the only consumer of queuing
-         * functionality. This will enable a specific Queuing mechanism per
-         * TransferType (see github issue #137).
+         * TODO Queuing responsibility should be moved to Project Negotiation,
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete the task associated to this TODO comment. ([squid:S1135](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1135))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=210&issues=AWVWjH94lRPS_ea-bThc&open=AWVWjH94lRPS_ea-bThc)

[ISSUE_KEY]: AWVWjH94lRPS_ea-bThc